content/courses/intro-to-ada/chapters/tasking.rst

```diff
@@ -195,27 +200,28 @@ In this example, we're making the task :ada:`T` wait one second after each
 time it displays the "hello" message. In addition, the main task is waiting
 1.5 seconds before displaying its own "hello" message
 
-Synchronization: rendez-vous
+Synchronization: rendezvous
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 The only type of synchronization we've seen so far is the one that happens
-automatically at the end of the main task. You can also define custom
+automatically at the end of a master construct with a subtask.
+You can also define custom
 synchronization points using the keyword :ada:`entry`. An *entry* can be
-viewed as a special kind of subprogram, which is called by the master task
+viewed as a special kind of subprogram, which is called by another task
 using a similar syntax, as we will see later.
 
-In the task definition, you define which part of the task will accept the
+In the task body definition, you define which part of the task will accept the
 entries by using the keyword :ada:`accept`. A task proceeds until it
-reaches an :ada:`accept` statement and then waits for the master task to
+reaches an :ada:`accept` statement and then waits for some other task to
 synchronize with it. Specifically,
 
-- The subtask waits at that point (in the :ada:`accept` statement),
+- The task with the entry waits at that point (in the :ada:`accept` statement),
   ready to accept a call to the corresponding entry from the master task.
 
-- The master task calls the task entry, in a manner similar to a procedure
-  call, to synchronize with the subtask.
+- The other task calls the task entry, in a manner similar to a procedure
+  call, to synchronize with it.
```

Comment:
"it" can be confusing in this context. We should be more specific (as in the original version).

Reply:
I originally said "... synchronize with the task with the entry" but that seemed quite redundant.  Feel free to rephrase it if you can find a reasonable way to do so.

Reply:
In this case, I prefer to be redundant. Also, the paragraph starts with "Specifically," so we can be more specific here...